tests/operator/helper.go

```diff
@@ -20,7 +20,12 @@ Package operator provides CNFCERT tests used to validate operator CNF facets.
 
 package operator
 
-import "strings"
+import (
+	"strings"
+
+	v1 "github.com/operator-framework/api/pkg/operators/v1"
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
```

Comment:
Do the `InstallMode` variables not exist in `v1`?  That way we don't have multiple imports?

Reply:
No, InstallMode does not exist.